assignments/closure.js

```diff
@@ -1,11 +1,34 @@
 // ==== Challenge 1: Write your own closure ====
 // Write a simple closure of your own creation.  Keep it simple!
-
+const name = 'Lyle Dylandy';
+function enroll()
+{
```

Comment:
This curly brace should be on line 4. This looks like C language style, JS has a few different conventions for code style. Use VS Code with the Prettier extension for automatic formatting on save, this will make your code easier for others (and yourself) to read.